android/app/src/main/java/org/fossasia/openevent/fragments/AboutFragment.java

```diff
@@ -122,9 +120,12 @@ protected int getLayoutResource() {
     @Override
     public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         super.onViewCreated(view, savedInstanceState);
-
-        event = realmRepo.getEvent();
-        event.addChangeListener(realmModel -> loadEvent(event));
+        aboutFragmentViewModel.getEvent().observe(this, eventData -> {
+            if (eventData != null) {
+                event = eventData;
+                event.addChangeListener(realmModel -> loadEvent(event));
+            }
+        });
```

Comment:
Simply don't send data from ViewModel if it is null. Keep activity and fragments free of `if`s and `for`s